src/resources/listings.js

```diff
@@ -21,7 +21,7 @@ class Listings extends ResourceBase{
       address: address,
       ipfsHash: ipfsHash,
       sellerAddress: contractData[0],
-      priceWei: contractData[2],
+      priceWei: contractData[2].toString(),
```

Comment:
@DanielVF I'm confused - why are we returning `priceWei` as a string but `price` as a number?.

Reply:
👍gotcha